### ISSUE TITLE
Running "Test" causes game to see wheel correctly 

### ISSUE THREAD

**Issue by elite4jonny** [berarma/oversteer#214]
Game is Euro Truck 2 (Free demo on steam)

In Linux native, game requires me to run

Evdev-joystick --e /dev --d 0 --f 0 
In order to correct a deadzone problem. Others online say they use proton instead for better performance (DirectX mode).

Running game in DirectX mode it's like the wheel (Thrustmaster T300) is always fully turned to the right hand side. Opening Oversteer before the game doesn't correct the behaviour. Opening Oversteer and running the wheel test, then opening the game causes the buttons on the wheel to be remapped, but it can be used correctly and the wheel input registers as expected. Closing oversteer or rebooting the computer causes the game to again fully steer to the right, and opening Oversteer and running the test again corrected the behaviour.

What is the test doing to remap the controls, shouldn't this be happening as soon as the software is started?

**Comment by elite4jonny** [comment]
I agree, absolutely unexpected behaviour but appears to be reproducible. Is
there any troubleshooting I could do, a before/after log of the inputs I
could find somewhere, because somehow oversteer is fixing whatever bug this
is!

**Comment by berarma** [comment]
You could try changing range, gain and autocenter values from Oversteer and see if those changes have any effect.

**Comment by Kimplul** [comment]
Thanks for the ping, took me a few days to find a bit of time to look into this but I can replicate, specifically the DX11 version of the game seems to behave this way. Adding `open_mode=0` to modprobe seems to fix this, could be that it's related to a comment @berarma made a few years ago by now: https://github.com/Kimplul/hid-tmff2/issues/56#issuecomment-1422552984

I'll play around a bit more and check if it's tied to some specific Proton version, not entirely sure what the best fix would be but one option could be to set `open_mode=0` by default in the driver. This would have the slight downside that the wheel's autocalibration throws it off center when plugged in, but I guess that's not too bad.
